File: src/main/java/org/homework20250105/java/animal/Human.java
package org.homework20250105.java.animal;

import org.homework20250105.java.animal.base.Animal;
import org.homework20250105.java.animal.base.TwoFeet;

public class Human extends Animal implements TwoFeet {

        public Human(String name, Integer age) {
        super(name, age, AnimalEnum.HUMAN);
    }

    @Override
    public void say() {
        System.out.println("Hello");
    }
}

File: src/main/java/org/homework20250105/kotlin/animal/Human.kt
package org.homework20250105.kotlin.animal

import org.homework20250105.kotlin.animal.base.Animal
import org.homework20250105.kotlin.animal.base.TwoFeet

class Human(name: String, age: Int?) : Animal(name, age, AnimalEnum.HUMAN), TwoFeet {
    override fun say() {
        println("Hello")
    }
}
